File: forward-system-frontend/src/core/dto/DistributionItemLogDto.ts
import type {UserDto} from "@/core/dto/UserDto.ts";

export interface DistributionItemLogDto {
    id: number,
    userId: number,
    user: UserDto,
    statusType: string,
    statusTypeRus: string,
    waitStart: string,
    waitUntil: string,
    createdAt: string
}

File: src/main/java/by/forward/forward_system/core/dto/messenger/v3/DistributionItemLogDto.java
package by.forward.forward_system.core.dto.messenger.v3;

import by.forward.forward_system.core.dto.rest.users.UserDto;
import by.forward.forward_system.core.enums.DistributionItemStatusType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DistributionItemLogDto {
    private Long id;
    private Long userId;
    private UserDto user;
    private DistributionItemStatusType statusType;
    private String statusTypeRus;
    private String waitStart;
    private String waitUntil;
    private String createdAt;
}
